### PR TITLE
added reviewers Instructions morse code

### LINKED ISSUE #77: Create "##Instructions for reviewer" for project Assertive Programming Kata
Please add a section to the bottom of this project's _index.md file.
The section heading should be EXACTLY "## Instructions for reviewer"

Basically we have to set reviewers up for success by telling them what success looks like and what to pay attention to when reviewing this project submission.

Description:

- [ ]  Look at a bunch of old reviews for this project for each cohort

 

- [ ] Find the common feedback [This should help you get an idea of what instructions to create]

- [ ]  Write them out clearly

- [ ]  The section heading should be EXACTLY "## Instructions for reviewer"

### PR DESCRIPTION
**Pull request by kaleem99**
Related issues: [please specify]

## Description:

What are you up to? Fill us in :)

## I solemnly swear that:

- [x] I ran the hugo server and looked at my changed in the browser with my own eyes
- [x] I ran the linter and there were no errors
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

**Review by EckardBerry — CHANGES_REQUESTED** [review]
Hello

Just want to make sure, when you say a "Global object" is used for alphabet characters, numbers etc.  I see most people declare a global constant, is it different when you do this in JS?  Do you declare a global object or a global constant in JS?

This will have an influence on a recruits' understanding if it is different for Java, Python and JS.

**Comment by kaleem99** [comment]
Global object but it should be of const type, should change the wording to global constant.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

File: content/projects/morse-code/_index.md [hunk 1]
```diff
@@ -57,4 +58,22 @@ For a problem like this one, unit tests are more appropriate.
...
+- Make sure that the assertions do not print an error if empty strings are passed in both functions, e.g 
+
+```
+lettersToMorseCode("") // shouldn't print an assertion failure
+morseCodeToLetters("") // shouldn't print an assertion failure
+```
+
+## Mark as excellent if:
+
+- An assertion function was used to assert if both functions input and output data are the same
```
Comment (AxelBankuna) [inline content/projects/morse-code/_index.md]: We've been requesting this as a compulsory requirement. Do we now consider this use of an assert function as optional?

File: content/projects/morse-code/_index.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ flavours:
 - any_language
 prerequisites:
   hard:
-  - topics/intro-to-asserive-programming
+  - topics/intro-to-assertive-programming
```
Comment (sheenarbw) [inline content/projects/morse-code/_index.md:8]: This is now a broken link.

If I ask you to open a file named "intro-to-assertive-programming" and yet the file doesn't exist then you wont be able to open the file.
If you update a link anywhere then you need to make sure it points to something that exists
If you change a file path then you need to make sure that everything that refers to that file is updated with the new path

This stuff is connected.

File: content/projects/morse-code/_index.md [hunk 3]
```diff
@@ -57,4 +58,19 @@ For a problem like this one, unit tests are more appropriate.
 
 This is optional.
 
-Add unit tests to your code. Leave the assertions in there just so that we can see that you know how to hit the requirements we laid out. But if you have some passing tests as well that would be sweet.
\ No newline at end of file
+Add unit tests to your code. Leave the assertions in there just so that we can see that you know how to hit the requirements we laid out. But if you have some passing tests as well that would be sweet.
+
+## Instructions for Reviewers
+
+- Ensure that the lettersToMorseCode function can convert numbers, special characters and alphabets to morseCode and the morseCodeToLetters function can convert morse code to numbers, alphabets and special characters.
```
Comment (sheenarbw) [inline content/projects/morse-code/_index.md]: rather say ...numbers, special characters and letters of the alphabet.
An alphabet is something that contains a lot of different letters. 

Also look over capitalisation.

And inline code can be surrounded by single backtickls to make it appear as code. Eg `lettersToMorseCode`

File: content/projects/morse-code/_index.md [hunk 4]
```diff
@@ -57,4 +58,19 @@ For a problem like this one, unit tests are more appropriate.
...
+- Make sure that both functions returns and not prints the output.
```
Comment (sheenarbw) [inline content/projects/morse-code/_index.md]: Make sure that both functions return the output. Neither function should print the output

File: content/projects/morse-code/_index.md [hunk 5]
```diff
@@ -57,4 +58,19 @@ For a problem like this one, unit tests are more appropriate.
...
+- When using assertions make sure that the assert statements are checking if the input and output lengths and input and output spaces are the same by checking if they are equal.
+- If Unit tests are used, make sure that there is a test that checks the input and output lengths and ensure that imports and exports was used correctly.
+- Make sure an assertion function was used to assert if both functions input and output data are the same
+- Make sure a global constant was used to contain all the data e.g alphabets, numbers, special characters and morse code.
```
Comment (sheenarbw) [inline content/projects/morse-code/_index.md]: letters of the alphabet

**Review by sheenarbw — CHANGES_REQUESTED** [review]
A few small things and one big thing

File: content/projects/morse-code/_index.md [hunk 6]
```diff
@@ -57,4 +58,19 @@ For a problem like this one, unit tests are more appropriate.
...
\ No newline at end of file
+Add unit tests to your code. Leave the assertions in there just so that we can see that you know how to hit the requirements we laid out. But if you have some passing tests as well that would be sweet.
+
+## Instructions for Reviewers
+
+- Ensure that the `lettersToMorseCode` function can convert numbers, special characters and letters of the alphabets to morse code and the `morseCodeToLetters` function can convert morse code to numbers, letters of the alphabets and special characters.
+- Make sure that both functions return the output. Neither function should print the output.
+- When using assertions make sure that the assert statements are checking if the input and output lengths and input and output spaces are the same by checking if they are equal.
+- If Unit tests are used, make sure that there is a test that checks the input and output lengths and ensure that imports and exports was used correctly.
+- Make sure an assertion function was used to assert if both functions input and output data are the same
```
Comment (AxelBankuna) [inline content/projects/morse-code/_index.md]: Missing full-stop at the end of this sentence.

**Review by AxelBankuna — APPROVED** [review]

**Review by Sbonelo01 — APPROVED** [review]
LGTM

**Review by sheenarbw — APPROVED** [review]

**Pull request merged**